ngraph/core/src/op/loop.cpp

```diff
@@ -399,3 +330,28 @@ bool op::v5::Loop::evaluate(const HostTensorVector& outputs, const HostTensorVec
         m_body, m_output_descriptions, m_input_descriptions, m_special_body_ports, outputs, inputs);
     return true;
 }
+
+void op::v5::Loop::clone(op::v5::Loop& dst, const OutputVector& new_args) const
```

Comment:
please rename from `clone` to `clone_to`

Reply:
Done